docs/UserGuide.md

```diff
@@ -227,6 +200,24 @@ Examples:
 * `addEvent n/Shoe Sale 30% d/30-05-2022 t/11:00 p/Discount on all shoes for up to 30%`
 * `addEvent n/Banana Discount 10% d/20-04-2022 t/14:00 p/10% discount on all bananas`
 
+### Editing an event : `editEvent`
+
+Edits an existing event in the application.
+
+Format: `editEvent INDEX [e/EVENT_TITLE] [d/DATE] [t/TIME] [p/PURPOSE]`
+
+- Edits the event at the specified `INDEX`. The index refers to the index number shown in the displayed event list.
+  The index must be **a positive integer** 1, 2, 3, …​, and it must be within the range of the event list index.
+- At least one of the optional fields must be provided.
```

Comment:
```suggestion
- Edits the event at the specified `INDEX`. The index refers to the index number shown in the displayed event list.
- The index must be **a positive integer** 1, 2, 3, …​, and it must be within the range of the event list index.
- At least one of the optional fields must be provided.
```
I think this needs a dash?

Reply:
I think it's supposed to be a complete paragraph since 'index' is like a local variable used for line 209 only and this sentence on 210 makes reference to it. The sentence on line 210 is supposed to be part of 209 but has moved to prevent checkstyle from throwing an error.